ontology/ntr_terms.py

```diff
@@ -580,4 +580,10 @@
         "organs": ["blood", "bodily fluid"],
         "systems": ["immune system"],
     },
+    "NTR:0000756": {
+        "cells": ["erythroid progenitor cell"],
+        "name": "Human Umbilical cord blood-Derived Erythroid Progenitor-2",
+        "organs": ["Umbilical cord"],
```

Comment:
The official UBERON term is not capitalized "U" so I think we should stick to that.